bin/machine-setup.sh

```diff
@@ -121,6 +121,17 @@ sudo systemctl start sol
 sudo systemctl enable sol
 sudo systemctl --no-pager status sol
 
+if [[ $NODE_TYPE = validator ]]; then
```

Comment:
Let's run this service on all node types (specifically api and warehouse need it too, but no harm having it everywhere)